packages/mui-material/src/Snackbar/Snackbar.d.ts

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { SxProps } from '@mui/system';
-import { ClickAwayListenerProps } from '@mui/base/ClickAwayListener';
+import { SnackbarUnstyledProps } from '@mui/base/SnackbarUnstyled';
```

Comment:
I wouldn't change the implementation in Material UI right away. As far as I remember we landed on adding a Snackbar component in the lab that we can use (overtime it would replace the Material UI implementation, but it would also give us time to find any potential problems with it).

Another thing, we decided to use the headless hooks inside the Material UI & Joy UI components, so I would recommend adding a hook first and using it in the new lab component.

Reply:
Reverted the changes in Material UI and added the hook.

As for the new Snackbar component in the lab, I will do it in the Stacked snackbars PR. Makes sense to do add and use it there.